src/server.c

```diff
@@ -5879,6 +5880,7 @@ sds genValkeyInfoString(dict *section_dict, int all_sections, int everything) {
                 "instantaneous_input_repl_kbps:%.2f\r\n", (float)getInstantaneousMetric(STATS_METRIC_NET_INPUT_REPLICATION) / 1024,
                 "instantaneous_output_repl_kbps:%.2f\r\n", (float)getInstantaneousMetric(STATS_METRIC_NET_OUTPUT_REPLICATION) / 1024,
                 "rejected_connections:%lld\r\n", server.stat_rejected_conn,
+                "total_sync_bypass_crc:%ld\r\n", server.stat_total_sync_bypass_crc,
```

Comment:
```suggestion
                "total_sync_bypass_crc:%ld\r\n", server.stat_total_sync_bypass_crc,
```
What is the use of this metric? What are end users supposed to do with this information?

Reply:
This metric is merely for testing purposes

Reply:
We generally prefer using log lines for testing purposes, since these info fields are part of our public facing contract and we can't change them.